src/background/global-context-factory.ts

```diff
@@ -26,6 +30,8 @@ export class GlobalContextFactory {
         telemetryDataFactory: TelemetryDataFactory,
         indexedDBInstance: IndexedDBAPI,
         persistedData: PersistedData,
+        issueFilingServiceProvider?: BugFilingServiceProvider,
```

Comment:
I think these parameters shouldn't be optional (the one call in production code passes them, they're skipped in test code - maybe the test should be more complete?).